Documentation/development-flow.md

```diff
@@ -277,9 +277,8 @@ git push origin HEAD
 
 #### Automated flow
 
-You will find a script at `contrib/backport_to_stable_branch.sh` at the root of the Rook repository.
-Execute it and it will do the necessary things and will push the backport branch.
-Then go on the Rook Github web page and create your pull request.
+If you want your pull request to be backported automatically, you can set the appropriate label on your pull request.
+For example, to backport your pull request on the `release-1.1` branch, you would need to set the `backport-release-1.1` label.
 
 #### Create the backport pull request
```

Comment:
Can we edit this whole section on manual flow and creating backports? Now mergify will be the only way we prefer to backport, correct? So we just need to instruct them to open a PR against master, then tag the PR for automatic backporting.

Reply:
I just thought there might be a case where someone needs to manually a backport but I can't find a use case so sounds good!